pkgs/development/python-modules/django-statici18n/default.nix

```diff
@@ -0,0 +1,18 @@
+{ stdenv, lib, buildPythonPackage, fetchPypi, django, django_appconf }:
+
+buildPythonPackage rec {
+  pname = "django-statici18n";
+  version = "2.0.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0cqwfirzjbanibq3mfz9lcwqnc8655zpysf9hk9g3lbwj2m478sp";
+  };
+
+  propagatedBuildInputs = [ django django_appconf ];
```

Comment:
Missing checkPhase and pythonImportsCheck.

Reply:
pypi package does not package test code